### PR TITLE
#28 new content of flag file

### LINKED ISSUE #28: modification in content of flag file
Flag file should contain:
- summary
- entry_title
- entry_id
- source
- data_center

### PR DESCRIPTION
**Pull request by aanersc**
close #28

### CODE REVIEW

File: geospaas_processing/copiers.py [hunk 1]
```diff
@@ -31,8 +31,13 @@ def write_flag_file(type_in_flag_file, source_path, dataset, destination_filenam
...
         with open(destination_filename + ".flag", "w") as flag_file:
             flag_file.write(f"type: {type_in_flag_file}" + os.linesep)
+            flag_file.write(f"entry_id: {dataset.entry_id}" + os.linesep)
+            flag_file.write(f"entry_title: {dataset.entry_title}" + os.linesep)
+            flag_file.write(f"source: {dataset.source}" + os.linesep)
+            flag_file.write(f"data_center: {dataset.data_center}" + os.linesep)
             for urlname in dataset.dataseturi_set.exclude(service=LOCAL_FILE_SERVICE):
-                flag_file.write(f"url: {urlname.uri}" + os.linesep)
+                flag_file.write(f"- url: {urlname.uri}" + os.linesep)
+            flag_file.write(f"summary: {dataset.summary}" + os.linesep)
```
Comment (aperrin66) [inline geospaas_processing/copiers.py:40]: Maybe build the string and then write it in one go, rather than calling `write()` multiple times.

Comment (aanersc) [reply geospaas_processing/copiers.py]: Done in 4b67c5b

File: geospaas_processing/copiers.py [hunk 2]
```diff
@@ -30,9 +30,15 @@ def write_flag_file(type_in_flag_file, source_path, dataset, destination_filenam
         writes the flag file in the case of request for it. ".flag" is the extension for flag file.
         """
         with open(destination_filename + ".flag", "w") as flag_file:
-            flag_file.write(f"type: {type_in_flag_file}" + os.linesep)
+            string_to_write = f"type: {type_in_flag_file}" + os.linesep
```
Comment (aperrin66) [inline geospaas_processing/copiers.py:33]: You already use f-strings, so you don't need to concatenate the line separator:
`f"type: {type_in_flag_file}{os.linesep}"`

Comment (aanersc) [reply geospaas_processing/copiers.py]: Done in 4a22268

**Review by aperrin66 — APPROVED** [review]

**Review by akorosov — APPROVED** [review]

**Pull request merged**